ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java

```diff
@@ -14,12 +14,6 @@
 package tech.pegasys.artemis.statetransition.util;
 
 import static java.lang.Math.toIntExact;
-import static tech.pegasys.artemis.datastructures.Constants.BASE_REWARD_QUOTIENT;
-import static tech.pegasys.artemis.datastructures.Constants.EPOCH_LENGTH;
```

Comment:
@schroedingerscode did you miss this? or am I missing something?

Reply:
Yep, I missed that.